sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient.cs

```diff
@@ -0,0 +1,969 @@
+        /// <param name="connectionId">Target connection Id</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> representing the result of the operation.</returns>
+        public virtual Response AddConnectionToGroup(string group, string connectionId, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.AddConnectionToGroup");
+            scope.Start();
+            try
+            {
+                return RestClient.AddConnectionToGroup(hub, group, connectionId, cancellationToken);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>Remove a user from all groups.</summary>
+        /// <param name="userId">Target user Id</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
+        public virtual async Task<Response> RemoveUserFromAllGroupsAsync(string userId, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.RemoveUserFromAllGroupsAsync");
+            scope.Start();
+            try
+            {
+                return await RestClient.RemoveUserFromAllGroupsAsync(hub, userId, cancellationToken).ConfigureAwait(false);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>Remove a user from all groups.</summary>
+        /// <param name="userId">Target user Id</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> representing the result of the operation.</returns>
+        public virtual Response RemoveUserFromAllGroups(string userId, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.RemoveUserFromAllGroups");
+            scope.Start();
+            try
+            {
+                return RestClient.RemoveUserFromAllGroups(hub, userId, cancellationToken);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>Grant permission to the connection.</summary>
+        /// <param name="permission">Current supported actions are joinLeaveGroup and sendToGroup.</param>
+        /// <param name="connectionId">Target connection Id.</param>
+        /// <param name="targetName">If not set, grant the permission to all the targets. If set, grant the permission to the specific target. The meaning of the target depends on the specific permission.</param>
```

Comment:
Since we only have two permission types, can we just say what this means for each?  I'd guess it was just the group, but this language is confusing enough that I'm not sure.

Reply:
I will do a docs pass after beta.